flask_server.py

```diff
@@ -1,10 +1,24 @@
-from Instastories import scrape_from_web, logToHTML
+from Instastories import scrape_from_web
 from flask import Flask, render_template, request
+import os
 app = Flask(__name__)
 
+COOKIE_PATH = "token.txt"
+
+def setLogFileList():
+    rendered_log = []
+    if not os.path.exists("run_history.log"):
+        f = open("run_history.log", "w").close()
+
+    with open("run_history.log", "r") as o:
+        for i in o.readlines():
+            rendered_log.append(str(i))
```

Comment:
I wouldn't call it a "rendered_log", it's just a list of line log. 
I'd change it as 

```
if not os.path.exists("run_history.log"):
    open("run_history.log", "w").close()

with open("run_history.log", "r") as f:
        return [log_line for for log_line in f.readlines()]
```

I'm not sure why you need to use str(i), but I didn't got to test it.

Reply:
Also change the name to getLogFileList, as you are mostly reading it.